XCUITests/CronTabsPerfTests.swift

```diff
@@ -0,0 +1,243 @@
+import XCTest
+
+class CronTabsPerformanceTest: BaseTestCase {
```

Comment:
Not sure if it will be better to have different classes for the different Performance tests (tabs, history, bookmarks...) or to have all the tests in the same one. We can leave it as it is know..but thinking ahead about this.